layers/+tools/transmission/packages.el

```diff
@@ -0,0 +1,64 @@
+;;; packages.el --- Transmission Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(setq transmission-packages
+      '(transmission))
+
+(defun transmission/init-transmission ()
+  (use-package transmission
+    :config (progn
```

Comment:
The package should be auto-loaded. That means using `:defer t`, and binding the entry points in `:init`. If the command `transmission` isn't auto-loaded, use the `:commands` directive to auto-load it.